gemfirexd/core/src/drda/java/com/pivotal/gemfirexd/internal/impl/drda/DRDAStatement.java

```diff
@@ -820,6 +847,12 @@ protected PreparedStatement prepare(String sqlStmt)   throws SQLException
 		  this.ctx.setSendSingleHopInformation(this.sendSingleHopInfo);
 		  ps = conn.prepareStatement(
                       sqlStmt, scrollType, concurType, withHoldCursor);
+			if (ps instanceof EmbedStatement) {
+				Properties props = new Properties();
+				props.setProperty(Attribute.USERNAME_ATTR, database.userId);
+				props.setProperty(Attribute.PASSWORD_ATTR, database.password);
+				((EmbedStatement)ps).setConnProps(props);
+			}
```

Comment:
Revert these changes. We are not supporting DRDA anymore (except for GemXD backward compatibility).